src/drt/README.md

```diff
@@ -17,15 +18,205 @@ guide format.
 
 ## Commands
 
-## Report wire length
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
+```
+
+### Detailed Route
+
+```tcl
+detailed_route 
+    [-output_maze filename]
+    [-output_drc filename]
+    [-output_cmap filename]
+    [-output_guide_coverage filename]
+    [-drc_report_iter_step step]
+    [-db_process_node name]
+    [-disable_via_gen]
+    [-droute_end_iter iter]
+    [-via_in_pin_bottom_layer layer]
+    [-via_in_pin_top_layer layer]
+    [-or_seed seed]
+    [-or_k_ k]
+    [-bottom_routing_layer layer]
+    [-top_routing_layer layer]
+    [-verbose level]
+    [-distributed]
+    [-remote_host rhost]
+    [-remote_port rport]
+    [-shared_volume vol]
+    [-cloud_size sz]
+    [-clean_patches]
+    [-no_pin_access]
+    [-min_access_points count]
+    [-save_guide_updates]
+    [-repair_pdn_vias layer]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-output_maze` | Path to output maze log file (e.g. `.output.maze.log`). |
+| `-output_drc` | Path to output DRC report file (e.g. `.output.drc.rpt`). |
+| `-output_cmap` | Path to output congestion map file (e.g. `.output.cmap`). |
```

Comment:
```suggestion
| `-output_maze` | Path to output maze log file (e.g. `output-maze.log`). |
| `-output_drc` | Path to output DRC report file (e.g. `output-drc.rpt`). |
| `-output_cmap` | Path to output congestion map file (e.g. `output.cmap`). |
```

Reply:
I was trying to convey the message that these could be the suffix of the filename. Example:

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/39641663/fad9daf5-ca87-4ff1-a11b-504244e764e7)

Reply:
Maybe `<NAME>.output.maze.log`. What do you think?